lib/Cookie/MetaCookie.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace Aerys\Cookie;
+
+class MetaCookie extends Cookie {
```

Comment:
This could be useful in amp/artax as well, maybe it would be nice to have an amp/http-message library in the future containing this code.